.circleci/test.sh

```diff
@@ -58,7 +58,16 @@ case ${CIRCLE_JOB} in
     RELEASE_WITH_EXPENSIVE_TESTS)
         run_unit_tests check_expensive
         ;;
-
+    COVERAGE)
+        cd "${GLOW_SRC}"
+        .circleci/run_coverage.sh
+        ;;
+    CHECK_CLANG_FORMAT)
+        CLANG_COMMAND=/usr/bin/clang-format-7
```

Comment:
where is it used?

Reply:
inside the format.sh, there's variable called CLANG_COMMAND